File: Easy/2413_Smallest_Even_Multiple/Solution.java



public class Solution {
    public int smallestEvenMultiple(int n) {
        if (n % 2 == 0) return n;
        return n * 2;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();

        // test cases 1
        System.out.println(solution.smallestEvenMultiple(5));  // Expected: 10
        
        // test cases 2
        System.out.println(solution.smallestEvenMultiple(6));  // Expected: 6
    }
}

File: Easy/2413_Smallest_Even_Multiple/Solution.py
from typing import List, Optional


class Solution:
    def smallestEvenMultiple(self, n: int) -> int:
        if n % 2 == 0:
            return n
            
        return n * 2


if __name__ == "__main__":
    solution = Solution()

    # test cases 1
    print(solution.smallestEvenMultiple(5))  # Expected: 10
    
    # test cases 2
    print(solution.smallestEvenMultiple(6))  # Expected: 6